quickwit/quickwit-common/src/tower/rate_estimator.rs

```diff
@@ -119,59 +124,57 @@ impl RateEstimator for SmaRateEstimator {
     fn update(&mut self, _started_at: Instant, ended_at: Instant, work: u64) {
```

Comment:
could we ditch `_start_at`? I know it is used in the dummy impl, but do we use it? Could it be internalized in the Dummy impl?

Reply:
I should actually use it... The argument should be used to check whether the duration spans multiple buckets, and if so, we should spread the amount of work evenly between buckets. However, in our case, the buckets are large (10s), and the requests are short, so the approximation seems ok. What do you think?